### PR TITLE
Ri program4

### PR DESCRIPTION
**Pull request by ritheesh4**

### CODE REVIEW

File: 00004_palindrome.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// 4. Enter a  string and find if it's a palindrome.
+
+const readLine = require('readline-sync');
+const inputString = readLine.question("Enter the string to check wether it is palindrome or not : ");
+
+let palindromeTesting = (inputString) => {
+    let reverseOfString = inputString.split('').reverse().join('');
+    if (inputString === reverseOfString){
+        console.log("The input string is a palindrome.");
+    }else {
```
Comment (kuism) [inline 00004_palindrome.js]: allignment

File: 00004_palindrome.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// 4. Enter a  string and find if it's a palindrome.
+
+const readLine = require('readline-sync');
+const inputString = readLine.question("Enter the string to check wether it is palindrome or not : ");
+
+let palindromeTesting = (inputString) => {
+    let reverseOfString = inputString.split('').reverse().join('');
```
Comment (kuism) [inline 00004_palindrome.js:6]: return the output string

File: 00004_palindrome.js (hunk 1 above)
Comment (ritheesh4) [reply 00004_palindrome.js]: OK.

File: 00004_palindrome.js (hunk 2 above)
Comment (ritheesh4) [reply 00004_palindrome.js:6]: Removed console and returned output.

**Pull request merged**